packages/clay-multi-select/src/htmlEscape.js

```diff
@@ -0,0 +1,13 @@
+const ampersandRegExp = /&/g;
+const lessThanRegExp = /</g;
+const greaterThanRegExp = />/g;
+const doubleQuoteRegExp = /"/g;
+const singleQuoteRegExp = /'/g;
+
+export const htmlEscape = string =>
```

Comment:
hey @wincent I didn't make many changes so in the end I keep the same thing.

Reply:
Hey @matuzalemsteles, we're out for the day, so I think it should be okay to merge and release. We can go over this tomorrow if there are changes needed.

Reply:
lg2m in any case. 👍